tests/page-objects/common/common.po.js

```diff
@@ -151,6 +152,11 @@ module.exports = {
     hamburgerMenu.click();
     helper.waitUntilReady(hamburgerMenuOptions);
   },
+
+  deleteRecord: async () => {
```

Comment:
I think this should be called `confirmDelete` or something - reading the code I was trying to work out why you were deleting twice!